extension/src/test/suite/vscode/quickPick.test.ts

```diff
@@ -161,7 +161,7 @@ suite('Quick Pick Test Suite', () => {
       const result = await resultPromise
 
       expect(result).to.deep.equal([5, 2, 1])
-    }).timeout(8000)
+    }).timeout(10000)
```

Comment:
Increased the timeout again, but I wonder if there is an issue in `selectMultipleQuickPickItems` that I could be missing: 

```javascript
const toggleQuickPickItem = async (number: number, itemsLength: number) => {
  for (let itemInd = 1; itemInd <= itemsLength; itemInd++) {
    await commands.executeCommand('workbench.action.quickOpenSelectNext')

    if (itemInd === number) {
      await commands.executeCommand('workbench.action.quickPickManyToggle')
    }
  }
}

export const selectMultipleQuickPickItems = async (
  numbers: number[],
  itemsLength: number,
  acceptItems = true
) => {
  for (const number of numbers) {
    await toggleQuickPickItem(number, itemsLength)
  }
  if (acceptItems) {
    return commands.executeCommand(
      'workbench.action.acceptSelectedQuickOpenItem'
    )
  }
}
```

Reply:
I would wait for an update event each time you run `await commands.executeCommand('workbench.action.quickPickManyToggle')` otherwise I think the results will always be unstable

Reply:
> I would wait for an update event each time you run await commands.executeCommand('workbench.action.quickPickManyToggle') otherwise I think the results will always be unstable

Good point, updated the code to wait for an update event when `commands.executeCommand('workbench.action.quickPickManyToggle')` is called!